t/helper/test-bloom.c

```diff
@@ -0,0 +1,89 @@
+/*
+ * test-bloom.c: test code to exercise bloom.c
+ */
+
+#include "test-tool.h"
+#include "git-compat-util.h"
+#include "bloom.h"
+#include "test-tool.h"
+#include "cache.h"
+#include "commit-graph.h"
+#include "commit.h"
+#include "config.h"
+#include "object-store.h"
+#include "object.h"
+#include "repository.h"
+#include "tree.h"
+
+struct bloom_filter_settings settings = DEFAULT_BLOOM_FILTER_SETTINGS;
+
+static void print_bloom_filter(struct bloom_filter *filter) {
+	int i;
+
+	if (!filter) {
+		printf("No filter.\n");
+		return;
+	}
+	printf("Filter:");
```

Comment:
the `\n` strings would be better at the _end_ of the strings instead of the beginnings. Allows dropping the final `printf("\n");`.